atom/browser/api/atom_api_app.cc

```diff
@@ -268,6 +269,30 @@ v8::Local<v8::Value> App::DefaultSession(v8::Isolate* isolate) {
     return v8::Local<v8::Value>::New(isolate, default_session_);
 }
 
+bool App::MakeSingleInstance(v8::Local<v8::Function> callback) {
```

Comment:
Hmm that's strange, the callback stores the V8 function internally so as long as the callback is available the V8 function should have been referenced, you can probably push your code so we can see what's going wrong.